src/main/java/org/reactome/release/dataexport/DataExporterStep.java

```diff
@@ -78,10 +85,10 @@ public void executeStep(Properties props) throws Exception {
 	 * @return Driver for the graph database being run by the Neo4J server
 	 */
 	private static Driver getGraphDBDriver(Properties props) {
-		String host = props.getProperty("host","localhost");
-		String port = props.getProperty("port", Integer.toString(7687));
-		String user = props.getProperty("user", "neo4j");
-		String password = props.getProperty("password", "root");
+		String host = props.getProperty("neo4jHostName","localhost");
+		String port = props.getProperty("neo4jPort", Integer.toString(7687));
+		String user = props.getProperty("neo4jUserName", "neo4j");
+		String password = props.getProperty("neo4jPassword", "root");
```

Comment:
Are these property names consistent with those used in other projects, or release-common-lib?

Reply:
I don't know - the only projects that I know of that also use the GraphDB are managed by EBI.  

Are there property names in release-common-lib for the GraphDB?  I don't think I've seen them, but if there are or we want to include them, we can standardize the property names I can adjust the names here or get a connection to the GraphDB through release-common-lib in later PRs.

Reply:
Hmmmm... then maybe for consistency we should use the property names used by EBI? I can't recall exactly what their property names for Neo4j connections look like.

Reply:
So far as I've been able to tell, they don't use properties files for configurations, but command-line arguments. 

Example: https://github.com/reactome-pwp/diagram-exporter/blob/master/src/main/java/org/reactome/server/tools/diagram/exporter/Main.java 

This will need to be addressed at some point, but I believe Joel gets to be a trendsetter here!

Reply:
Yay, I get to be a trendsetter!

I think I will leave the property names for connecting to a Neo4j graph db as they are here unless there are any suggestions for changes?

It probably does make sense to move the logic for connection to a graph db into release-common-lib at some point, though, since this will be a fairly common thing to do across Reactome projects.